addons/editor-dark-mode/addon.json

```
@@ -992,6 +992,32 @@
       "name": "3.Darker",
       "id": "3darker",
       "description": "A dark theme based on a userscript by infinitytec",
+      "values": {
+        "page": "#111111",
+        "primary": "#855cd6",
+        "highlightText": "#ccb3ff",
+        "menuBar": "#202020",
+        "popup": "#000a",
+        "activeTab": "#202020",
+        "tab": "#151515",
+        "selector": "#202020",
+        "selector2": "#202020",
+        "selectorSelection": "#111111",
+        "accent": "#151515",
+        "input": "#202020",
+        "workspace": "#151515",
+        "categoryMenu": "#202020",
+        "palette": "#202020cc",
+        "fullscreen": "#000000",
+        "stageHeader": "#000000",
+        "border": "#ffffff0d",
+        "dots": false
+      }
+    },
+    {
+      "name": "3.Darker Blue",
```

```suggestion
      "name": "3.Darker (blue)",
```
(for consistency with "Scratch's default colors (blue)")